translator/cmdutil/translatorutil.go

```diff
@@ -224,7 +225,32 @@ func TranslateJsonMapToYamlConfig(jsonConfigValue interface{}) (interface{}, err
 	if err = conf.Marshal(cfg); err != nil {
 		return nil, err
 	}
-	return conf.ToStringMap(), nil
+	strMap := conf.ToStringMap()
+	RemoveTLSRedacted(strMap)
+	return strMap, nil
+}
+
+// TODO: Switch to a custom translator approach at some point
+func RemoveTLSRedacted(stringMap map[string]interface{}) {
```

Comment:
Curious -- how did we handle this for the metrics forwarding tls configurations previously? Did it not crash then?